Integrations/IntezerV2/IntezerV2.py

```diff
@@ -0,0 +1,251 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+
+''' IMPORTS '''
+
+import requests
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+API_KEY = demisto.getParam('APIKey')
+SERVER_URL = 'https://analyze.intezer.com/api'
+API_VERSION = '/v2-0'
+BASE_URL = SERVER_URL + API_VERSION
+IS_AVAILABLE_URL = 'is-available'
+ERROR_PREFIX = 'Error from Intezer:'
+ACCEPTABLE_HTTP_CODES = {200, 201, 202}
+USE_SSL = not demisto.params().get('insecure', False)
+
+http_status_to_error_massage = {
+    400: '400 Bad Request - Wrong or invalid parameters',
+    401: '401 Unauthorized - Wrong or invalid api key',
+    403: '403 Forbidden - The account is not allowed to preform this task',
+    404: '404 Not Found - Analysis was not found',
+    410: '410 Gone - Analysis no longer exists in the service',
+    500: '500 Internal Server Error - Internal error',
+    503: '503 Service Unavailable'
+}
+dbot_score_by_verdict = {
+    'malicious': 3,
+    'suspicious': 2,
+    'trusted': 1,
+    'neutral': 1,
+    'no_threats': 1
+}
+
+''' HELPER FUNCTIONS '''
+
+
+def handle_response(response, acceptable_http_status_codes):
+    if response.status_code not in acceptable_http_status_codes:
+        error_msg = http_status_to_error_massage.get(response.status_code, "Failed to perform request")
+        return_error(f'{ERROR_PREFIX} {error_msg}')
+
+    return response.json()
```

Comment:
wrap this with try and except exception for failed to parse json, and return appropriate error